build-image-src/Dockerfile-dotnet8

```diff
@@ -0,0 +1,100 @@
+ARG IMAGE_ARCH
+FROM public.ecr.aws/lambda/dotnet:8-preview-$IMAGE_ARCH
```

Comment:
We should remove `preview` tag once OCI image reaches GA.

Reply:
It's removed now